features/step_definitions/auction_create_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Given(/^there is a closed auction$/) do
-  @auction = FactoryGirl.create(:auction, :closed, :with_bids)
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :c2_budget_approved)
```

Comment:
So, I decided I'd rather not put this in the Factory Girl traits, because I don't like the traits having too many undeclared side-effects. And then this isn't really something that comes up much in the unit and controller tests. So I really only needed to assert this here, where we are looking at the page elements in an integration test